src/table/Root.tsx

```diff
@@ -8,15 +8,16 @@ import TableWrapper from './components/TableWrapper';
 import { TableContextProvider } from './context';
 import muiSetup from './mui-setup';
 import { RenderProps, TableWrapperProps } from './types';
+import { Row } from '../types';
 
 export function render(props: RenderProps, reactRoot?: ReactDom.Root) {
-  const { direction, selectionsAPI } = props;
+  const { direction, selectionsAPI, tableData } = props;
   const muiTheme = muiSetup(direction);
 
   reactRoot?.render(
     <StyleSheetManager stylisPlugins={direction === 'rtl' ? [rtlPluginSc] : undefined}>
       <ThemeProvider theme={muiTheme}>
-        <TableContextProvider selectionsAPI={selectionsAPI}>
+        <TableContextProvider selectionsAPI={selectionsAPI} tableRows={tableData?.rows as Row[]}>
```

Comment:
We know that rows exist here, it just has to be optional on the type, to work with the native implementation

Reply:
Hmm, not sure what is best practice here. The thing is that we have two root files that, one for web implementation, one for react native (mobile app). Nebula has constraints on what some functions must be called etc in the index files. So we have to override the root file with root.native. They have to share types, so tableData has to be optional. BUT, it is always defined when THIS code runs. I think casting here indicates that we actually have a non-empty array, where `|| []` might be confusing to someone not knowing about this, since tableRows will never be `[]`